client/utils/auth.py

```diff
@@ -61,12 +66,23 @@ def make_refresh_post(refresh_token):
     client.request_token(refresh_token=refresh_token, **params)
     return client.access_token, client.expires_in
 
+
 def get_storage():
     with open(REFRESH_FILE, 'rb') as fp:
         storage = pickle.load(fp)
-    return storage['access_token'], storage['expires_at'], storage['refresh_token']
+
+    access_token = storage['access_token']
+    expires_at = storage['expires_at']
+    refresh_token = storage['refresh_token']
+
+    return access_token, expires_at, refresh_token
+
 
 def update_storage(access_token, expires_in, refresh_token):
+    if not access_token and expires_in and refresh_token:
```

Comment:
I think you want 

    if not (access_token and expires_in and refresh_token):